src/abstractdataframe/iteration.jl

```diff
@@ -55,10 +62,6 @@ end
 Return a `DFColumnIterator` that iterates an `AbstractDataFrame` column by column.
 Iteration returns a tuple consisting of column name and column vector.
 
-`DFColumnIterator` has a custom implementation of the `map` function which
```

Comment:
This feature sounds useful, why remove it? People can use comprehensions if they want an array.